menu/fsck2025.json

```diff
@@ -0,0 +1,23 @@
+
+{
+	"version": 2025042800,
+	"url": "https://cfp.ctbk.de/fsck-2025/schedule.xml",
+	"title": "FSCK 2025",
+	"start": "2025-05-09",
+	"end": "2025-05-11",
+	"timezone": "Europe/Berlin",
+	"metadata": {
+		"links": [
+			{ "url": "https://wiki.ctbk.de/FSCK2025", "title": "Website" },
+			{ "url": "https://wiki.ctbk.de/FSCK2025/Hotels", "title": "Where to stay in Backnang" },
```

Comment:
Given it's a sidebar menu entry, I'd suggest shortening it, maybe to "Where to stay".

Reply:
Yeah, could you change that please? Well spotted @Zverik I should make a check for that I suppose.

Reply:
Did that. Sorry for the delay